src/ArcxAnalyticsSdk.ts

```diff
@@ -253,6 +258,19 @@ export class ArcxAnalyticsSdk {
     return true
   }
 
+  private _trackClicks() {
+    window.addEventListener('click', (event) => {
+      this._onClick(event)
+    })
+  }
+
+  private _onClick(event: any) {
+    this.event(CLICK_EVENT, {
+      elementId: getElementIdentifier(event),
+      content: event.target.textContent,
+    })
```

Comment:
@stuart-watt is this enough to be able to understand on what page this particular click event happened?

Reply:
@gtupak I'm not sure what is happening here. From #66 i see that you are capturing the type of click with `elementId` and the content of the HTML botton object thingo. I don't think that information is enough to tell us on what page the click is happening, but we can simply order the events and look for the PAGE event that immediately precedes the button click. So that information would be inferred by the clickstream journey, rather than the CLICK event itself.